app/views/unattended/provisioning_templates/finish/preseed_default_finish.erb

```diff
@@ -54,6 +54,12 @@ echo 'root:<%= root_pass -%>' | /usr/sbin/chpasswd -e
 <%= snippet "blacklist_kernel_modules" %>
 
 <%= snippet_if_exists(template_name + " custom post") -%>
+
+<% unless host_param('kernel_parameters').nil? -%>
```

Comment:
Is it intentional that this also handles empty strings or is that an oversight?

Reply:
I believe having no kernel parameters is a valid setting

Reply:
True, but is the default that there are parameters and an empty string would wipe it out? I don't know the answer here, just figuring out if you thought about that or not.